airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/pipeline.py

```diff
@@ -190,6 +191,20 @@ def prepare_container_for_poe_tasks(
     # Set working directory to the poetry package directory
     container = container.with_workdir(f"/airbyte/{poetry_package_path}")
 
+    # If a package from `airbyte-platform-internal` is required, modify the entry in pyproject.toml to use https instead of ssh, when run in Github Actions
```

Comment:
A simpler but hackier option would be to download the private package from the private repo outside of airbyte-ci in the GHA workflow and edit the `pyproject.toml` to point to a local path instead of using a git URL. 

The best approach would be to host the private packages in a private PyPi repo.

Reply:
I'll look into moving it into the GHA when I'm wrapping up the GHA-specific PR. 

Agree re hosting the private package, but it didn't seem worth it for just this one thing.